OneSignalSDK/onesignal/src/main/java/com/onesignal/OSPreferencesWrapper.java

```diff
@@ -0,0 +1,81 @@
+package com.onesignal;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.Set;
+
+class OSPreferencesWrapper implements OSSharedPreferences {
```

Comment:
Lets just use the whole thing `OSPreferencesWrapper` should be `OSSharedPreferencesWrapper`